files/build_templates/pcie-mon.timer

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Monitor the PCIe device status periodically every 1 min
```

Comment:
This service is now one-time check at boot, and is no longer monitoring periodically, correct?